iroh-rpc/src/client.rs

```diff
@@ -442,14 +443,15 @@ mod test {
         let size = 4_048;
         let chunk_size = 1_024;
         // make request
-        let mut s = a_client
+        let s = a_client
             .streaming_call("b", "stream", StreamRequest { size, chunk_size })
             .await
             .expect("call to client b failed");
 
         let mut res = Vec::new();
 
         let mut num_chunks = 0;
+        pin_mut!(s);
```

Comment:
The `async_stream` docs mention this requirement. I don't understand how to eloquently pin the stream in the `streaming_call` function, so that users of this library won't have to call this themselves. But not sure how.

Reply:
requiring this for the moment is fine